src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-wed calendar--month-29">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--wed calendar--month-length--29">
```
Not critical, but pay attention to the task description
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/0a9cba24-f61b-46c2-ba06-e1990080152e)